### ISSUE TITLE
How to realize a company relationship chart ?

### ISSUE THREAD

**Issue by Goallying** [AAChartModel/AAChartKit#1153]
![src=http___cdn jiemodui com_img_Public_Uploads_item_20171219_1513622290976615 png refer=http___cdn jiemodui](https://user-images.githubusercontent.com/12438493/113374795-8e840800-93a0-11eb-973b-5f6c769ddac7.jpg)

**Comment by AAChartModel** [comment]
AAChartKit do not support it

**Comment by AAChartModel** [comment]
Maybe the apache Echarts can help you 

https://echarts.apache.org/v4/examples/zh/editor.html?c=graph

![image](https://user-images.githubusercontent.com/16357599/113404925-8e552e00-93db-11eb-8cb5-ae638605ff9b.png)
![image](https://user-images.githubusercontent.com/16357599/113405071-c9eff800-93db-11eb-89d5-0893fde27b2b.png)

**Comment by Goallying** [comment]
it seems does't support on iOS.

**Comment by AAChartModel** [comment]
You can load the ECharts with WKWebView on iOS.

**Comment by Goallying** [comment]
yeah,I download a sample HTML file and using WKWebView ,but nothing displaying. I have no experience working on it........

**Comment by Goallying** [comment]
<img width="505" alt="企业微信截图_044edaaf-1ac7-41a2-86d0-3a276cc660d6" src="https://user-images.githubusercontent.com/12438493/113644966-8a196100-96b8-11eb-9eaa-993e95355645.png">
